tests/jerry/fail/regression-test-issue-2774.js

```diff
@@ -0,0 +1,135 @@
+// Copyright JS Foundation and other contributors, http://js.foundation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/* Discard the output of the 'print' function */
+print = function () {}
```

Comment:
Why? I think we should use `assert` in this file

Reply:
We could change all the `print` function calls to `assert`s in the test case which was generated by fuzzer but unfortunately most of the checks are not valid, so it would fail instantly. I'd not like to bother the original test case since it revealed an issue with the string/memory management not the correctness of the `JSON.stringify` routine.